_includes/comparison.html

```diff
@@ -22,7 +22,7 @@ <h2 class="section-heading--primary"> Service Mesh Comparison </h2>
         <td>1.9</td>
         <td>1.4</td>
         <td>1.1</td>
-        <td>0.7</td>
+        <td>0.82</td>
```

Comment:
For simplicity reasons, we decided to show only major and minor version in the table. But, I'll simply remove the "2" after merging :)